pkg/device42/client.go

```diff
@@ -0,0 +1,36 @@
+package device42
```

Comment:
Suggestion: the standard repo layout we've defined is the package name should match the domain, and the implementation should be in the name of the specific files, e.g.:

```
|- pkg/
    |- domain/
        |- ipamfetcher.go
    |- ipamfetcher/
        |- device42_client.go
...
```